src/binary/ibc-setup/utils/get-default-home-path.spec.ts

```diff
@@ -0,0 +1,26 @@
+import test from 'ava';
+
+import { getDefaultHomePath } from './get-default-home-path';
+
+const processEnvCopy = { ...process.env };
```

Comment:
Yes, that's why I make a copy first and then before each test I "reset" it:
```
test.beforeEach(() => {
  process.env = processEnvCopy;
});
```

Reply:
Right, but what about after the last test?

Reply:
No need to reset it after the last test because
>AVA is highly opinionated and runs tests concurrently, with a separate process for each test file.

Or do I understand it wrong?

Reply:
Ah great, thanks ava.